tensorboard/webapp/notification_center/types.ts

```diff
@@ -0,0 +1,25 @@
+/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+import {Action, Store} from '@ngrx/store';
+import {State} from '../app_state';
+
+/**
+ * A notification chunk displayed in the notification center.
+ */
+export type Notification = {
+  date: string;
```

Comment:
date being a string is weird. I think it is more natural to have `number` that describes milliseconds since epoch or a `Date` object

Reply:
Good suggestion. Done.